secboot/secboot_sb.go

```diff
@@ -64,6 +65,28 @@ func LockSealedKeys() error {
 	return lockTPMSealedKeys()
 }
 
+func copyKeyring(devPath, legacyDevPath string) error {
```

Comment:
this probably needs a comment explaining what it is doing, also probably a different name that is closer to explain the purpose?

Reply:
Changed it and added explanation.